isos/bootstrap/bootstrap

```diff
@@ -76,11 +76,30 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     mkdir -p ${MOUNTPOINT}/usr/lib
     cp -r /usr/lib/iptables ${MOUNTPOINT}/usr/lib/
 
+    mkdir -m 755 -p ${MOUNTPOINT}/.tether/etc ${MOUNTPOINT}/.tether/lib/modules ${MOUNTPOINT}/.tether/usr/lib/iptables
+    mkdir -m 555 -p ${MOUNTPOINT}/.tether/proc ${MOUNTPOINT}/.tether/sys
+
+    touch ${MOUNTPOINT}/.tether/etc/hostname ${MOUNTPOINT}/.tether/etc/hosts ${MOUNTPOINT}/.tether/etc/resolv.conf
```

Comment:
Curious:  Is it necessary to touch these files in `${MOUNTPOINT}/.tether`?  I know they need to be created on scratch, but doesn't the tether create them here automatically?